.github/workflows/narwhal.yml

```diff
@@ -26,8 +26,8 @@ concurrency:
   cancel-in-progress: ${{ github.ref != 'refs/heads/main' }}
 
 env:
-  # Swap between Narwhal and Mysticeti each epoch.
-  CONSENSUS: "swap_each_epoch"
+  # Test Narwhal consensus as long as Narwhal runs in mainnet.
```

Comment:
Does this mean we will no longer have nightly mysticeti runs?

Reply:
The nightly run switching between epochs is going to be in `Simtest Nightly` instead of `Simtest Nightly - Mysticeti`. The nightly run with Narwhal is going to be in `Simtest Nightly - Narwhal` instead of `Simtest Nightly`

Reply:
I think there is still value keeping a run with Mysticeti enabled only . How can we ensure that mysticeti will run for all the simulation tests (unless all of them span across multiple epochs which I don't remember)

Reply:
Just to make sure, there are 3 configurations:
- Narwhal: running in mainnet, so CI coverage is needed.
- Swap each epoch: will run in devnet and testnet, so CI coverage is needed.
- Mysticeti: do you mean we need to add a 3rd CI configuration? I believe it is a bit of overkill.

Reply:
But we can definitely add one.

Reply:
+1 to having a 3rd workflow for just mysticeti

Reply:
it is indeed a bit of overkill but especially with the velocity of the features we are adding we might need some extra safety. If it slows down things a lot then we can remove it. what do you think?

Reply:
Added a nightly Mysticeti nightly workflow. I think pretty soon after testnet we can switch to only Mysticeti in tests and devnet as well.